guide/14-deep-learning/object-detection.ipynb

```diff
@@ -431,7 +431,8 @@
    "source": [
```

Comment:
<ul><li>multiple object tracking in video? this can be added as:-</li></ul><p><em>Multiple-object tracking on a video </em>can be performed using<em> </em><code>predict_video</code>&nbsp;function in the&nbsp;<code>arcgis.learn</code>&nbsp;module.</p><p> </p><ul><li>spellcheck in "To <strong>neable </strong>tracking, set the&nbsp;<code>track</code>&nbsp;parameter in.." Should be <strong>enable</strong></li></ul><p> </p><ul><li><em> </em>Where are the tracking options available, is it in the predict video function? Then rephrase :-</li></ul><p><em>The following options/parameters are available in the predict video function for the user to decide:-</em></p><p> </p><ul><li>rephrase</li></ul><p><code><em>vanish_frames</em></code><em>&nbsp;i.e. the number of frames the object remains absent from the frame to be considered as vanished.</em></p><p><code><em>detect_frames</em></code><em>&nbsp;i.e. the number of frames an object remains present in the frame to start tracking.</em></p><p><code><em>assignment_iou_thrd</em></code><em>&nbsp;i.e. There might be multiple trackers detecting and tracking objects. The Intersection over Union (iou) threshold can be set to assign a tracker with the mentioned threshold value.</em></p><p> </p><ul><li>repetition of words. Can be rephrased</li></ul><p><em>Object tracking in&nbsp;</em><code><em>arcgis.learn</em></code><em>&nbsp;is performed using&nbsp;</em><strong><em>Kalman-filtering and Hungarian Assignment Algorithm</em></strong><em>. It can be used as follows:</em></p><p> </p><p>The rest of the technical definition is not clear from this description. If scope permits these could be elaborated further.</p><p> </p><ul><li>rephrase</li></ul><p>&nbsp;<em>match these bounding box with former bounding boxes we have been using</em></p><p> </p><p>To make this sound more coherent, Could you please explain, how does Kalman filter and Hungarian Algorithm fits in in arcgis.learn implementation part.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>